docs/choose-content-topic.md

```diff
@@ -1,30 +0,0 @@
----
-title: How to Choose a Content Topic
----
-
-# How to Choose a Content Topic
```

Comment:
As mentioned, I think we need a section for content topic. Maybe more focus on "How is content addressed on Waku" which introduce content topic, explain how to choose one and also mention the different with pubsub topic.

Pubsub topic = network/routing
content topic = application

And the separation of both domain enabling more flexibility on dials such as scalability and privacy

Reply:
i was hoping for the "how-to"s to be a different section "guides"

I could maybe add a theoretical brief on the subject and link to the guide within

Reply:
sounds good